bigbluebutton-html5/private/locales/en.json

```diff
@@ -398,6 +398,8 @@
     "app.audioModal.settingsTitle": "Change your audio settings",
     "app.audioModal.helpTitle": "There was an issue with your media devices",
     "app.audioModal.helpText": "Did you give permission for access to your microphone?  Note that a dialog should appear when you try to join audio, asking for your media device permissions, please accept that in order to join the audio conference.  If that is not the case, try changing your microphone permissions in your browser's settings.",
+    "app.audioModal.help.noSSL": "The webpage must use HTTPS for microphone access to work. Please contact the administrator of your website",
+    "app.audioModal.help.macNotAllowed": "Check in your Mac system preferences > Security & Privacy > Privacy > Microphone, that it's checked.",
```

Comment:
"It looks like your Mac System Preferences are blocking access to your microphone. Open System Preferences > Security & Privacy > Privacy > Microphone, and verify that the browser you're using is checked."